pkg/reconciler/taskrun/taskrun_test.go

```diff
@@ -1660,14 +1660,14 @@ status:
   - reason: ToBeRetried
     status: Unknown
     type: Succeeded
-    message: "Provided results don't match declared results; may be invalid JSON or missing result declaration:  \"aResult\": task result is expected to be \"array\" type but was initialized to a different type \"string\""
+    message: "provided results don't match declared results; may be invalid JSON or missing result declaration:  \"aResult\": task result is expected to be \"array\" type but was initialized to a different type \"string\""
   sideCars:
   retriesStatus:
   - conditions:
     - reason: TaskRunValidationFailed
       status: "False"
       type: "Succeeded"
-      message: "Provided results don't match declared results; may be invalid JSON or missing result declaration:  \"aResult\": task result is expected to be \"array\" type but was initialized to a different type \"string\""
+      message: "provided results don't match declared results; may be invalid JSON or missing result declaration:  \"aResult\": task result is expected to be \"array\" type but was initialized to a different type \"string\""
```

Comment:
This might actually be the ones we want to keep capitalized since users would just read through the retriesStatus message.

Reply:
This error message is returned from [here](https://github.com/tektoncd/pipeline/blob/7039282b54df6a69931f9f40957848ec0eacc764/pkg/reconciler/taskrun/validate_taskrun.go#L254). Is this only returned for retriesStatus or is that just for this test case? I can update the error message and the test case to be capitalized again. I just wanted to make sure this wasn't also returned somewhere else where the capitalization could be incorrect style.

Reply:
Thanks for looking into this, Matthew! Your question makes sense. 
The `message: "Provided results...."` we are seeing is user-facing. 

The issue seems to be that we haven't wrapped the error being returned well enough with a capitalized error message in [taskrun.go](https://github.com/tektoncd/pipeline/blob/7069889e796b0b12e12a3d1095a5f1a3d98c2739/pkg/reconciler/taskrun/taskrun.go#L566).

So the workaround for this PR here might be just to restore this change and in the future we might want to keep the error messages wrapped so such issue wouldn't recur:/

Reply:
Thank you for the clarification Jerome. I have reverted that change.